docs/app/Examples/elements/Input/States/index.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react'
+import ComponentExample from 'docs/app/Components/ComponentDoc/ComponentExample'
+import ExampleSection from 'docs/app/Components/ComponentDoc/ExampleSection'
+
+import { Message } from 'src'
+
+const InputStates = () => (
```

Comment:
In order to aide in searching, let's always keep the component name in the default export name.  Let's also keep the filename the same as the default export (except for index.js files of course).

This way the default export matches the filename and we can easily search for either.  This makes automated refactors much easier to work with.  Once we're done with the legacy components I plan to update all doc files to be consistent.